scripts/context_retrieval/wrangle_llama_index_data.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    for split in split_to_dataframe.keys():
</pre><p>using a loop for just two keys is pretty verbose. Why not just write a function.</p><p>database_df["text_vector"] = get_vectors_from_disc(split: "database")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>